WordPressKitTests/AccountSettingsRemoteTests.swift

```diff
@@ -62,6 +62,7 @@ class AccountSettingsRemoteTests: RemoteTestCase, RESTTestable {
         stubRemoteResponse(meSettingsEndpoint, filename: getAccountSettingsSuccessMockFilename, contentType: .ApplicationJSON)
         remote.getSettings(success: { settings in
             XCTAssertEqual(settings.username, self.username, "The usernames should be equal.")
+            XCTAssertEqual(settings.usernameCanBeChanged, false, "The usernames should be equal.")
```

Comment:
Was the message in this line a copy/paste from the line above? Is there something more descriptive we could add?